generators/constants.js

```diff
@@ -42,8 +42,8 @@ module.exports = {
     },
     versions: {
         micronaut: '2.1.1',
-        micronautData: '2.0.0',
-        micronautOpenApi: '2.1.0',
+        micronautData: '2.1.0',
```

Comment:
IMO the micronaut-data should be aligned with the one of the micronaut BOM we use.

Because our micronaut-data version is only used for the annotation processor, but not for the dependency (at least for maven): https://github.com/jhipster/generator-jhipster-micronaut/blob/main/generators/server/templates/pom.xml.ejs#L393

Otherwise we use two different micronaut-data versions.

Reply:
Totally right, didn't think of that. So we need to wait or should we include the micronaut-data-bom additonally?

Reply:
We could explicitly override the version in the pom / build files if we want to use a version that is newer than the one in the Micronaut BOM

Reply:
Micronaut 2.1.2 BOM has been released, including updated micronaut-data dependency. So you could update update it too, and both versions will be aligned 😃

Reply:
:heavy_check_mark: Done.